### DISCUSSION TITLE
aesmd service start faied

### DISCUSSION THREAD

**Discussion by reclock** [gramineproject/gramine#1535] [Q&A]
ERROR:systemctl:the ExecStartPre control process exited with error code 

I have installed the Pccs service and the AESM service. The PCCS service started normally, but the AESMD service failed to start (Docker environment, os: ubuntu22.04). The AESM service status only shows the failure without specific error information.

I have run through the SGXDataCenterAttention Primitives/SampleCode/QuoteGenerationSample example normally, and there is no problem with the sgx device.

![图片](https://github.com/gramineproject/gramine/assets/23696606/94788055-1c10-4f97-92fb-e8d74ca04324)

**Comment by reclock** [comment]
this is my lib
![图片](https://github.com/gramineproject/gramine/assets/23696606/d66cb4ea-e2d9-45a2-95ca-33f718c75c3a)

**Comment by mkow** [comment]
This problem is unrelated to Gramine, please ask PCCS / AESM maintainers for support. We aren't developing nor maintaining these programs.
